internal/clients/ec2/ec2_client.go

```diff
@@ -34,11 +34,13 @@ func NewEC2Client(ctx context.Context, params ...string) *Client {
 	cache := aws.NewCredentialsCache(credentials.NewStaticCredentialsProvider(
 		config.AWS.Key, config.AWS.Secret, config.AWS.Session))
 
-	region := params[0]
-	if region == "" {
-		region = config.AWS.Region
+	region := config.AWS.Region
+	if params != nil {
```

Comment:
```suggestion
	if params != nil && params[0] != "" {
```

Reply:
And then remove the fallback ofc...

Reply:
This does not fix it either, guys, I highly recommend not to use variadic functions. See below :-)